kahuna/public/js/services/api/collections-api.js

```diff
@@ -127,6 +127,35 @@ collectionsApi.factory('collections',
             });
     }
 
+    function findCollectionResource(image, collectionToMatch){
+        return image.data.collections.filter(collection => {
```

Comment:
Decided not to use 'find' in previous PR. I have given the humble filter function a honest name.